src/clj/rems/guide_utils.clj

```diff
@@ -11,21 +10,18 @@
         meta (assoc meta
                     :file (common-util/normalize-file-path (:file meta))
                     :doc (-> &env :ns :doc))]
-    `(rems.guide-functions/render-namespace-info
+    `(render-namespace-info
       ~name
       ~meta)))
 
-(comment
-  (meta (find-ns 'rems.guide-macros)))
-
 (defmacro component-info [component]
   ;; TODO we'd like to use normalize-file-path on the component
   ;; metadata at compile time, just like we do for namespaces above.
   ;; However it seems that there's no way to get the metadata at
   ;; compile time (e.g. resolve and ns-resolve return nil). Thus we
-  ;; normalize the path at runtime in rems.guide-functions/link-to-source.
+  ;; normalize the path at runtime in link-to-source.
   `(let [m# (meta (var ~component))]
-     (rems.guide-functions/render-component-info
+     (render-component-info
```

Comment:
Hmm so this works these days?

Reply:
It's just normal macro behaviour where the vars get qualified with the current namespace name, which happens to be the name of the corresponding cljs namespace where these functions are defined...

Reply:
Yeah, you moved these into the same ns though not to the same file.